src/metrics/include/metrics/MetricsReporter.hpp

```diff
@@ -235,6 +242,75 @@ class ReporterT final {
         }
     }
 
+    void reportCedarCsv(std::ostream& out,
+                        long long systemTime,
+                        long long metricsTime,
+                        v1::Permission perm) const {
+        out << "Clocks" << std::endl;
+        out << "clock,nanoseconds" << std::endl;
+        writeClocks(out, systemTime, metricsTime);
+        out << std::endl;
+
+        // We use an ordered map here to avoid defining a custom hash function for
+        // std::pair<std::string, std::string>. There aren't likely to be many (Actor, Operation)
+        // combinations for this to matter too much in terms of efficiency.
+        auto opThreadCounts = std::map<std::pair<std::string, std::string>, size_t>{};
+        out << "OperationThreadCounts" << std::endl;
+        out << "actor,operation,workers" << std::endl;
+        for (const auto& [actorName, opsByType] : _registry->getOps(perm)) {
+            for (const auto& [opName, opsByThread] : opsByType) {
+                if (actorName == "Genny" &&
+                    (opName == "ActorStarted" || opName == "ActorFinished")) {
+                    // The cedar-csv metrics format ignores the Genny.ActorStarted and
+                    // Genny.ActorFinished operations reported by the DefaultDriver because the
+                    // OperationThreadCounts section effectively tracks the number of concurrent
+                    // actors and that number isn't meaningfully changing over time.
+                    continue;
+                }
+
+                opThreadCounts[std::make_pair(actorName, opName)] += opsByThread.size();
+            }
+        }
+        for (const auto& [key, count] : opThreadCounts) {
+            const auto& [actorName, opName] = key;
+            out << actorName << ",";
+            out << opName << ",";
+            out << count << std::endl;
+        }
+        out << std::endl;
+
+        out << "Operations" << std::endl;
+        out << "timestamp,actor,thread,operation,duration,outcome,n,ops,errors,size" << std::endl;
+        for (const auto& [actorName, opsByType] : _registry->getOps(perm)) {
+            for (const auto& [opName, opsByThread] : opsByType) {
+                if (actorName == "Genny" &&
+                    (opName == "ActorStarted" || opName == "ActorFinished")) {
```

Comment:
consider putting this repeated `if` statement into a function `if skipReporting(actorName, opName)`

Reply:
Done.